EventsExpress/ClientApp/src/actions/events/filter/actions.js

```diff
@@ -0,0 +1,19 @@
+import { stringify } from 'query-string';
+import { get_events } from '../../event/event-list-action';
+
+export const applyFilters = filters => {
+    return async dispatch => {
+        const filter = stringify({
+            ...filters,
```

Comment:
That is some hacky stuff here. Let's not do the spread operator at all. Just fill in correct properties.

Reply:
On backend we are using owners term as event owners and on frontend I used term organizers to match the requirements, that's why I am mapping organizers to owners here. Locally, I removed spread operator and changed only properties, but it's also not good because filters, passed to this action, are form values and filters.organizers = undefined will cause errors. I'm doing this because in query string we want to see only owners in this case. Another variant in this situation is to map organizers to owners on backend instead of mapping on frontend. Will it be better than current solution?

Reply:
Much better, at least it is understandable what you are doing with this version.